packages/trip-form/src/MetroModeSelector/AdvancedModeSettingsButton/index.tsx

```diff
@@ -130,7 +143,7 @@ const AdvancedModeSettingsButton = ({
           {modeButton.enabled && <Check2 />}
         </label>
       </StyledModeSettingsButton>
-      {modeButton.modeSettings.length > 0 && (
+      {containsSubsettings && (
         <AnimateHeight duration={300} height={modeButton.enabled ? "auto" : 0}>
```

Comment:
Add `aria-expanded={modebutton.enabled}` to this element. Maybe add a `aria-describedby` or `aria-details` to the checkbox that points to this element, in addition or in place of `subsettingsStatusLabel`.